src/packer.pkr.hcl

```
@@ -124,19 +124,20 @@ build {
   sources = ["source.amazon-ebs.freeipa"]
 
   provisioner "ansible" {
-    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
-    playbook_file   = "src/upgrade.yml"
+    playbook_file = "src/upgrade.yml"
+    use_sftp      = true
```

I saw no indication that this should be removed.

```suggestion
    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
    playbook_file   = "src/upgrade.yml"
    use_sftp        = true
```

---

That extra argument was at one time necessary, but [for about a year Ansible has been setting `ansible_python_interpreter=auto` by default](https://github.com/ansible/ansible/blob/6059116a64afd4d53d82f846f9c3342ab21f9b7c/lib/ansible/config/base.yml#L1512-L1529).  (The code change was made sixteen months ago, but I'm not sure exactly under which release it first emerged.)

---

Ok. Since there was no explanation for the change in this repo's commits I just wanted to make sure this was intentional.